ejc-sql.el

```diff
@@ -333,25 +339,32 @@ Prepare buffer to operate as `ejc-sql-mode' buffer."
       (setq ejc-result-file-path
             (ejc--get-result-file-path))))
 
-(defun ejc-eval-org-snippet (body params)
+(defun ejc-eval-org-snippet (&optional orig-fun body params)
   "Used to eval SQL code in `org-mode' code snippets."
-  (let* ((beg (save-excursion
-                (goto-char (nth 5 (org-babel-get-src-block-info)))
-                (end-of-line)
-                (right-char 1)
-                (point)))
-         (end (save-excursion
-                (goto-char beg)
-                (+ beg (length body) (skip-chars-forward "\t ")))))
-    (ejc-eval-user-sql-at-point
-     :beg beg
-     :end end
-     :sync ejc-org-mode-show-results
-     :display-result (not ejc-org-mode-show-results))
-    (if ejc-org-mode-show-results
-        (with-temp-buffer
-          (insert-file-contents (ejc-get-result-file-path))
-          (buffer-string)))))
+  (if (or (not ejc-org-mode-babel-wrapper)
+          (and (cdr (assq :engine params))
+               (not
+                (yes-or-no-p
+                 (concat "ejc-sql is enabled, ignore source block :engine parameters\n"
```

Comment:
Actually, I'd prefer my version of the message here since a user have to pass a set of params to execute queries.
`:engine` - is only one of it. Furthermore, `\n` is redundant here. I split code for 2 lines to improve readability.
```lisp
(concat "ejc-sql is enabled, ignore source block connection"
        " parameters and use ejc-sql to execute it? ")
```
Or probably this version (see: https://orgmode.org/worg/org-contrib/babel/languages/ob-doc-sql.html):
```lisp
(concat "ejc-sql is enabled, ignore source block connection"
        " header arguments and use ejc-sql to execute it? ")
```